PaginationTracker.podspec

```diff
@@ -31,8 +31,6 @@ Pod::Spec.new do |s|
 	# Core spec
 	s.subspec 'Core' do |ss|
 		ss.source_files = 'Sources/Core/**/*.swift'
-
-		ss.dependency 'Alamofire', '~> 5.4'
```

Comment:
That's not going to work, you depend on it in `AsyncPaginatedListRepository.swift`.

Honestly can't remember "why" we need the dependency 🤔